src/stories/Quiz/Component/QuestionTypes/CoverQuestion.stories.tsx

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { expect } from '@storybook/jest';
+import { CoverQuestion } from '@constructor-io/constructorio-client-javascript/lib/types';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { within } from '@storybook/testing-library';
+import CoverTypeQuestion from '../../../../components/CoverTypeQuestion/CoverTypeQuestion';
+import { getMockQuestionWithImage, getMockQuestion } from '../../tests/mocks';
+import QuestionTypeDecorator from './QuestionTypeDecorator';
+import { QuestionTypes } from '../../../../components/CioQuiz/actions';
+
+const coverQuestionWithImage = getMockQuestionWithImage(QuestionTypes.Cover);
+const coverQuestionWithoutImage = getMockQuestion(QuestionTypes.Cover);
+
+const meta: Meta<typeof CoverTypeQuestion> = {
+  title: 'Quiz/Questions/CoverQuestion',
+  component: CoverTypeQuestion,
+  argTypes: {},
+};
+
+export default meta;
+type Story = StoryObj<typeof CoverTypeQuestion>;
+
+export const CoverQuestionWithImage: Story = {
+  args: {},
+  decorators: [(story) => QuestionTypeDecorator(story, coverQuestionWithImage as CoverQuestion)],
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+    document.querySelector('.cio-container--with-image');
+
+    // 👇 Assert DOM structure
+    const title = canvas.getByText('This is question title');
+    const description = canvas.getByText('This is question description');
+    const quizImage = document.querySelector('.cio-question-image');
+    await expect(quizImage).toBeInTheDocument();
+    await expect(title).toBeInTheDocument();
+    await expect(description).toBeInTheDocument();
+  },
+};
+
+export const CoverQuestionWithoutImage: Story = {
```

Comment:
Is this expected to look like this 🤔 
![image](https://github.com/Constructor-io/constructorio-ui-quizzes/assets/2954126/58c3122c-bb9e-47c9-bf3a-020f89987171)

Reply:
Do we want to fix it only in the stories. Or fix it in the component code itself (this might create side effects).
Why I'm asking this is that this is how they would look to people importing the component so do we need to fix that @esezen

Reply:
and same for other components with off styles

Reply:
Discussed on Slack